method/CalculateWordsInString.java

```diff
@@ -0,0 +1,35 @@
+package method;
+
+// Ունեք սովորական String փոփոխական, որի մեջ անգլերեն տառերով տեքստա գրած պրոբելներով։։ Դուք պիտի գրեք մեթոդ,
+// որը կստանաս տվյալ String-ը որպես պարամետր ու կվերադարձնի String-ում բառերի քանակը։
+public class CalculateWordsInString {
+    public static void main(String[] args) {
+        String Example = "  Բարև ախպեր ջան, ներողություն     տունը   մեկը չիգա։    ";
+        int result = calculateWords(Example);
+        System.out.println(result);
+    }
+
+    static int calculateWords(String str) {
+        int quantity = 0;
+
+        // This fragment of code checking if there is 'space' in the beginning of the string
+        int k = 0;
+        while (str.charAt(k) == 32) {
```

Comment:
1/please use ' ' instead of 32
2. please do alt+ctrl+l
3. are negative numbers prime?

Reply:
fixed